lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -185,10 +185,14 @@ public Task cancelClaim(String taskId)
   @Override
   public Task forceCancelClaim(String taskId)
       throws TaskNotFoundException,
-          InvalidOwnerException,
-          NotAuthorizedOnWorkbasketException,
-          InvalidTaskStateException {
-    return this.cancelClaim(taskId, true);
+          InvalidTaskStateException,
+          InvalidOwnerException
+      try {
+          return this.cancelClaim(taskId, true);
+      } catch (InvalidOwnerException e) {
+          throw new SystemException(
+            "this should not have happened. You've discovered a new bug!", e);
+      }
   }
```

Comment:
This is not compiling. Did you properly set up your dev environment?
It should look like this:
```
  @Override
  public Task forceCancelClaim(String taskId)
      throws TaskNotFoundException, InvalidTaskStateException, NotAuthorizedOnWorkbasketException {
    try {
      return this.cancelClaim(taskId, true);
    } catch (InvalidOwnerException e) {
      throw new SystemException("this should not have happened. You've discovered a new bug!", e);
    }
  }
```

Reply:
Not sure, I downloaded the repo but when compiling always get many many errors so I must be doing something wrong. I changed the code to look exactly as you suggested. Let me know if it still is not working on your end.

Reply:
Pipeline is working correctly now :-) thank you